client/dashboard/utils.js

```diff
@@ -36,6 +36,35 @@ export function getCurrencyRegion( countryState ) {
 	return region;
 }
 
+/**
+ * Gets the product IDs for items selected for purchase.
```

Comment:
Nice - thanks for moving this 👍

Small nitpick: might update this description to read:

```suggestion
 * Gets the product IDs for items based on the product types and theme selected in the onboarding profiler.
```

Reply:
Fixed in https://github.com/woocommerce/woocommerce-admin/pull/3444/commits/d7762abe8ad6717ef3db01792f1b1fe9f3584906.